examples/json-csv/json_csv.bal

```diff
@@ -5,8 +5,8 @@ import ballerina/log;
 // Return the result as a tuple, which contains the headers and fields.
 function getFields(map<json> rec) returns [string[], string[]] {
     string[] fields = [];
-    foreach var field in rec {
-        fields[fields.length()] = field.toString();
+    foreach var recordField in rec {
+        fields[fields.length()] = recordField.toString();
     }
```

Comment:
You can do the following to get an array of string values of the values in the map:
```ballerina
string[] fields = m.'map((val) => val.toString()).toArray();
```